app/controllers/membership_applications_controller.rb

```diff
@@ -126,20 +138,32 @@ def authorize_membership_application
   end
 
 
-  def new_upload_file(upload_file_param)
-    if upload_file_param['actual_files']
-      upload_file_param['actual_files'].each do |upload_file|
+  def new_file_uploaded(params)
+
+    successful = true
+
+    if params.fetch('uploaded_file', false) && params['uploaded_file'].fetch('actual_files', false)
+
+      params['uploaded_file']['actual_files'].each do |uploaded_file|
```

Comment:
I thought that I'd have to explicitly check to see if `params['uploaded_file']['actual_files']` existed.  I thought I'd have to check or else I'd get an error.

Now I've learned that doing it that way will be ok; it _won't_ blow up if a key doesn't exist.  

Reviews are so valuable. :-)

I'll make the changes and you two can see how it looks.